.github/actions/changes/readme.md

```diff
@@ -0,0 +1,55 @@
+# Docker Hub Login and Image Signing #
```

Comment:
I want to see docs like this for every action we write.   Make future user/devs (even if ourselves) hate us less.